include/datatypes.h

```diff
@@ -139,16 +141,6 @@ typedef struct {
     uint8_t             *keyblob;
 } ehsm_keyblob_t;
 
-//aes
-typedef struct
-{
-    uint32_t    ciphertext_size;
-    uint32_t    aad_size;
-    uint8_t     iv[SGX_AESGCM_IV_SIZE];
-    uint8_t     mac[SGX_AESGCM_MAC_SIZE];
-    uint8_t     payload[];   /* ciphertext + aad */
-} aes_gcm_key_data_t;
-
 //rsa
```

Comment:
you didn't change the definition of the structure? 

I think the first important thing is to define the structure, coding is just the pipeline work! if the structure not ready you may update this PR again and again...

Reply:
Here, for rsa, it will use the same struct as aes_gcm encryption.
The rsa keypair will be storage into payload[]